scripts-available/CDB_CartodbfyTable.sql

```diff
@@ -1234,7 +1234,7 @@ BEGIN
     AND c.oid = reloid
     AND am.amname != 'gist'
   LOOP
-    sql := Format('CREATE INDEX %s_%s_gix ON %s USING GIST (%s)', relname, rec.attname, reloid::text, rec.attname);
+    sql := Format('CREATE INDEX ON %s USING GIST (%s)', reloid::text, rec.attname);
```

Comment:
:-) Thanks for the explanation. :ship: it!

On Thu, Aug 27, 2015 at 4:48 PM, Paul Ramsey <notifications@github.com>
wrote:

> In scripts-available/CDB_CartodbfyTable.sql
> <https://github.com/CartoDB/cartodb-postgresql/pull/127#discussion_r38103736>
> :
>
> > @@ -1234,7 +1234,7 @@ BEGIN
> >      AND c.oid = reloid
> >      AND am.amname != 'gist'
> >    LOOP
> > -    sql := Format('CREATE INDEX %s_%s_gix ON %s USING GIST (%s)', relname, rec.attname, reloid::text, rec.attname);
> > +    sql := Format('CREATE INDEX ON %s USING GIST (%s)', reloid::text, rec.attname);
>
> It’s happening w/i the context of a loop that is finding only geometry
> columns that don’t already have gist indexes. So as long as the table
> rename retained the reference to the previous indexes, no extra indexes
> will get created.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CartoDB/cartodb-postgresql/pull/127/files#r38103736>.
>



-- 
Raul Ochoa
rochoa@cartodb.com